sys-libs/libnbd/libnbd-1.14.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit autotools bash-completion-r1 python-single-r1
+
+DESCRIPTION="NBD client library in userspace"
+HOMEPAGE="https://gitlab.com/nbdkit/libnbd"
+SRC_URI="https://download.libguestfs.org/libnbd/$(ver_cut 1-2)-stable/${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples fuse gnutls +uri-support go ocaml python test"
+
+RESTRICT="!test? ( test ) test? ( userpriv )"
```

Comment:
userpriv alone seems wrong here, if fuse tests are omitted without USE=fuse

Reply:
fix it